status/boost_check_library.py

```diff
@@ -83,6 +107,33 @@ def check_organization_include(self):
                 ''',
                 'org-inc-one')
     
+    def check_organization_macros(self):
+        libdir = os.path.join(self.library_dir,'include','boost',self.library_name)
+        if os.path.isdir(libdir):
```

Comment:
Why not just return? Reduces nesting

Reply:
None of the other checks early-out. I was matching the style of the original source code. Some people have strong feelings on early-out.

Reply:
Never understood the reasoning for opposing early out on checks like this, especially if the function returns nothing. It really hinders readability. And other functions do (kind of) early out: https://github.com/boostorg/boost/blob/e6a40e1d69b795654f02f940ebfe2543b8dc6ad9/status/boost_check_library.py#L294-L304

Counterexample is https://github.com/boostorg/boost/blob/e6a40e1d69b795654f02f940ebfe2543b8dc6ad9/status/boost_check_library.py#L333 but I'm not sure if this was decided or just happens to be the authors preference.